scworkercluster.js

```diff
@@ -70,6 +71,19 @@ process.on('message', function (masterMessage) {
     if (masterMessage.type == 'terminate' && masterMessage.data.killClusterMaster) {
       terminate();
     }
+    if (masterMessage.type == 'terminate' && forceKillTimeout) {      
+      childExitMessage = {}
+      setTimeout(function () {
+        for (var i in workers) { 
+          if (!childExitMessage[i]) {            
+            console.warn(forceKillTimeout + " ms no exit signal from Worker " + i + "(PID:"+workers[i].process.pid+') force kill it' );
+            //process.kill(workers[i].process.pid,'SIGINT')
+            process.kill(workers[i].process.pid,'SIGHUP') 
+            //process.kill(workers[i].process.pid,'SIGKILL')  
+          }
+        }
+      }, forceKillTimeout)
+    }
```

Comment:
There is a bunch of extra whitespace characters at the end of most lines.

Reply:
Maybe there should be an option in SC which lets you specify how to kill the workers (which specific signal to use) that way the user can choose.